core/src/main/java/nl/nn/adapterframework/lifecycle/ServletManager.java

```diff
@@ -155,4 +144,17 @@ public void register(DynamicRegistration.Servlet servlet) {
 
 		registerServlet(servlet.getName(), servlet.getServlet(), servlet.getUrlMapping(), servlet.getRoles(), servlet.loadOnStartUp(), parameters);
 	}
+
+	public static ServletSecurity.TransportGuarantee getTransportGuarantee(AppConstants appConstants, String propertyName) {
```

Comment:
Since AppConstants is a static instance you can just refer to it by calling AppConstants.getInstance()
Saves this additional parameter, which is always 'AppConstants.getInstance()'